### PR TITLE
Update sbt sonatype plugin to latest

### PR DESCRIPTION
**Pull request by tonytw1**
## What does this change?

Latest sonatype plug in.
Specifically the 3.9 series includes:

```
Add sonatypeTimeoutMillis to wait longer time until close/promote stage completes. The default is 60 minutes.
```

### CODE REVIEW

**Review by JustinPinner — APPROVED** [review]
LGTM 👍

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "15.9.11-SNAPSHOT"
```
Comment (frj) [inline version.sbt:1]: Should this be in this PR?

Comment (JustinPinner) [reply version.sbt:1]: I _think_ that update happens direct to master when the release is done?

Comment (tonytw1) [reply version.sbt:1]: Yeah - that's a mistake. Dirty workspace.
 Reverting.

**Pull request merged**